_docs/Qabel-Protocol-Box.md

```diff
@@ -281,6 +285,19 @@ url: STR // URL to the metadata file that contains information about the folder
 },
 ```
 
+ExternalFile:
+
+```
+{
+owner: STR, // public key of the owner of that File
+name: STR, // object name,
+size: LONG, // uncompressed file size
+mtime: LONG, // modification time as seconds since epoch
```

Comment:
How can you track mtime and size changes? They would only reach the FM wouldn't they?